homeassistant/components/sensor/waqi.py

```diff
@@ -77,7 +77,7 @@ def name(self):
         try:
             return self._details['city']['name']
```

Comment:
Other sensors show a specific measure (i.e. "Temperature" or "Humidity"). In your case it will be just name of the city. My original intention was that the name reads as "Air Quality Index at XYZ - ..." or WAQI XYZ for short :)